Author: Kedap <kedap.dev@protonmail.com>
Summary: Fix the problem with compilation in outdate versions of rust
--- a/src/core_funcions.rs	2021-07-29 14:53:40.409958616 -0500
+++ b/src/core_funcions.rs	2021-08-03 19:48:29.836559186 -0500
@@ -274,7 +274,7 @@
     let depen_arr = &adi_paquete.dependencias;
 
     //Selecionando los gestores instalados
-    for gestor in catalogo {
+    for gestor in catalogo.iter() {
         let comando = Command::new("bash")
             .arg("-c")
             .arg(gestor)
@@ -351,7 +351,7 @@
     let mut gestores = Vec::new();
 
     //Selecionando los gestores instalados
-    for gestor in catalogo {
+    for gestor in catalogo.iter() {
         let comando = Command::new("bash")
             .arg("-c")
             .arg(gestor)
